WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -147,11 +147,15 @@ public async Task<CoinJoinResult> StartCoinJoinAsync(Func<Task<IEnumerable<Smart
 			{
 				if (!IsRoundEconomic(roundParameters.MiningFeeRate, RoundStatusUpdater.CoinJoinFeeRateMedians, FeeRateMedianTimeFrame))
 				{
-					throw new CoinJoinClientException(CoinjoinError.UneconomicalRound, "Uneconomical round skipped.");
+					string roundSkippedMessage = "Uneconomical round skipped.";
+					currentRoundState.LogInfo(roundSkippedMessage);
```

Comment:
can remove the string, same as other log messages in coinjoinclient
```suggestion
					currentRoundState.LogInfo("Uneconomical round skipped.");
```